scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -48,12 +54,14 @@ namespace <%= namespace %>.SDK {
         public string ID = null;
         private string _VariantId;
         private long _Quantity;
+        private decimal _Price;
         private ObservableDictionary<string, string> _CustomAttributes;
         private OnCartLineItemChange OnChange;
-        
-        public CartLineItem(string variantId, OnCartLineItemChange onChange, long quantity = 1, IDictionary<string, string> customAttributes = null) {
+
+        public CartLineItem(string variantId, OnCartLineItemChange onChange, decimal price, long quantity = 1, IDictionary<string, string> customAttributes = null) {
```

Comment:
Yes this sounds good, I personally prefer we remove it entirely. And use 
`public void AddOrUpdate(ProductVariant variant, long? quantity = null, Dictionary<string, string> customAttributes = null) ` as the base.

To me a `CartLineItem` is a `ProductVariant` with a quantity. Extracting properties of `ProductVariant` to create a `CartLineItem` causes confusion and adds room for error.